### ISSUE TITLE
Add "Send Break" Feature

### ISSUE THREAD

**Issue by DjinnCPU** [microsoft/vscode-serial-monitor#94]
Hi!

I have been using your Serial Monitor extension for Visual Studio Code and found it very helpful. However, I recently needed to send a break condition over the serial connection, and it appears that this feature is not currently supported in the extension.

I would like to request the addition of a feature to send break conditions in the Serial Monitor extension. This could be implemented as a button in the Serial Monitor panel. Ideally, this would allow users to send a break condition without needing to switch to an external terminal application.

Thank you.

Best regards

**Comment by gcampbell-msft** [comment]
Hi, @DjinnCPU, thanks for your request!

A quick question, would toggling the DTR or RTS settings be able to accomplish these requests? They are located in the additional settings view. Please let me know if this works for you, if not, we'll investigate getting this request on the backlog.

**Comment by pvinb** [comment]
https://stackoverflow.com/questions/37540293/what-does-break-mean-in-real-terms-while-using-a-uart

**Comment by gcampbell-msft** [comment]
@pvinb Is this feature something you're still requesting? If so, I can make sure to re-open it and unmark it as stale. Please let me know, thanks!

**Comment by pvinb** [comment]
I try to do the switch to use vs code serial-monitor instead of using putty, tera term or realterm.
Some of our old products uses break to turn them on and off depending on the length/duration of the break.
For me, this is very low priority request, more of a nice to have feature. But to answer " would toggling the DTR or RTS settings be able to accomplish these requests? ", the answer is no.

**Comment by gcampbell-msft** [comment]
Got it. Thanks for the update. I'll re-open and put it on our backlog and we will keep it there until we get more customer votes indicating there is more customer demand for this. Thanks!

**Comment by hongshui3000** [comment]
https://www.decisivetactics.com/support/view?article=how-do-i-send-a-break  
I need this feature too. I guess it should be possible to send CTRL+BREAK in terminal mode?

**Comment by gcampbell-msft** [comment]
@hongshui3000 Please make sure to upvote the original post, thanks!

**Comment by srberard** [comment]
This is a feature I need as well.  Several of our devices can be reset via a break sequence.  That is extremely helpful for debugging/troubleshooting as it avoids having to power cycle the device manually.

**Comment by rumbledethumps** [comment]
Yes, please. I still use minicom only because I need to send breaks.
